evutil.c

```diff
@@ -352,13 +352,19 @@ evutil_win_socketpair(int family, int type, int protocol,
 		EVUTIL_SET_SOCKET_ERROR(WSAEAFNOSUPPORT);
 		return -1;
 	}
-	if (family == AF_UNIX && evutil_check_working_afunix_()) {
```

Comment:
Hm, so this changes the behavior, it forces AF_UNIX if it supported, why?

Reply:
Okay, overwriting family is fine, but the documentation for the `evutil_socketpair()` should reflect this cases, can you adjust it?